Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -4807,6 +4819,7 @@ void LocalFrameView::paintContents(GraphicsContext& context, const IntRect& dirt
     }
 
     didPaintContents(context, dirtyRect, paintingState);
+    didPaintSomeContents();
```

Comment:
Should `didPaintContents` calls `didPaintSomeContents` instead?

Reply:
That would result in double calls for the RenderView's layer.